README.md

```diff
@@ -73,11 +73,12 @@ substantial performance issue with the `gzip` feature. When rolling files
 it will zip log archives automatically. This is a problem when the log archives
```

Comment:
Only slightly ranting about GitHub reviews not being as good as GitLab reviews. It won't let me include the lines above this. How about some thing like this.

```
If you are using the file rotation in your configuration there us a known
substantial performance issue with either the `gzip` or `zstandard`
features. When rolling files it will zip log archives automatically. This is
a problem when the log archives are large as the zip process occurs in
the main thread and will halt the process until the zip process
completes.
```